src/gc/gc.d

```diff
@@ -384,7 +407,15 @@ struct GC
     }
 
     /**
+     * Returns a bit field representing all block attributes set for the memory
+     * referenced by p.
      *
+     * Params:
+     *  p = A pointer to the root of a valid memory block or to null.
```

Comment:
Please use `base` instead of `root` here and in the following comments. It might get confusing when talking about root pointers to life objects, e.g. in `addRoot`.